src/index.js

```diff
@@ -1,28 +1,191 @@
 import '@fortawesome/fontawesome-free/js/all.js';
-import { todoList, todoListElement } from './modules.js';
-
-import('./style.css');
-
-const todo = () => {
-  todoList.forEach((todo) => {
-    const { completed, description } = todo;
-    const todoElement = document.createElement('li');
-    let list = '';
-    list += `
-                    <div class='content'>
-                       <input type="checkbox" ${
-  completed ? 'checked' : ''
-} class='input' />
-                        <span>${description}</span>
-                    </div>
-                   <span class='icon'>
-                   <i class="fa-solid fa-ellipsis-vertical"></i>
-                   </span>
-                    `;
-    todoElement.innerHTML = list;
-    todoElement.classList.add('list-row');
-    todoListElement.appendChild(todoElement);
+
+import('./css/style.css');
+
+/* DOMContentLoaded */
+document.addEventListener('DOMContentLoaded', main);
+
+/* main() FUNCTION */
+
+function main() {
+  // get alltodos and initialise listeners
+  addTodo();
+  // dragover on .todos container
+  document.querySelector('.todos').addEventListener('dragover', function (e) {
+    e.preventDefault();
+    if (
+      !e.target.classList.contains('dragging') &&
+      e.target.classList.contains('card')
+    ) {
+      const draggingCard = document.querySelector('.dragging');
+      const cards = [...this.querySelectorAll('.card')];
+      const currPos = cards.indexOf(draggingCard);
+      const newPos = cards.indexOf(e.target);
+      console.log(currPos, newPos);
+      if (currPos > newPos) {
+        this.insertBefore(draggingCard, e.target);
+      } else {
+        this.insertBefore(draggingCard, e.target.nextSibling);
+      }
+      const todos = JSON.parse(localStorage.getItem('todos'));
+      const removed = todos.splice(currPos, 1);
+      todos.splice(newPos, 0, removed[0]);
+      localStorage.setItem('todos', JSON.stringify(todos));
+    }
+  });
+  // add new todos on user input
+  const add = document.getElementById('add-btn');
+  const txtInput = document.querySelector('.txt-input');
+  add.addEventListener('click', function () {
+    const item = txtInput.value.trim();
+    if (item) {
+      txtInput.value = '';
+      const todos = !localStorage.getItem('todos')
+        ? []
+        : JSON.parse(localStorage.getItem('todos'));
+      const currentTodo = {
+        item,
+        isCompleted: false,
+      };
+      addTodo([currentTodo]);
+      todos.push(currentTodo);
+      localStorage.setItem('todos', JSON.stringify(todos));
+    }
+    txtInput.focus();
```

Comment:
- Great job so far 👍 your site looks good and congrats on implementing the drag and drop feature 💯 but there are still some issues you'll need to address.
- [ ] After deleting a task, the `indexes` of the remaining tasks are to be updated. I noticed you used `id` so therefore the `ids` of the remaining tasks are to be updated after deleting some tasks. In the screenshot below I added 3 elements and I deleted the first element but the `ids` of the remaining elements did not update. Kindly address this issue.
- [ ] (OPTIONAL) Also after dragging and dropping an element, all the `ids` of the remaining elements are to be updated. It would be great if you could implement that.
- Screenshot:
![Capture](https://user-images.githubusercontent.com/85113567/154793055-cd1604b9-4ae0-4592-9f02-8d546dcadfac.PNG)
- The `ids` should always begin from 1, so if the first task is deleted, the new `id` of the second task should now be 1.